jas/common_test.go

```diff
@@ -232,10 +300,50 @@ func TestGetAnalyzerManagerXscEnvVars(t *testing.T) {
 			technologies:   []techutils.Technology{},
 			expectedOutput: map[string]string{utils.JfMsiEnvVariable: "msi"},
 		},
+		{
+			name:         "With git repo url",
+			msi:          "msi",
+			gitRepoUrl:   "gitRepoUrl",
+			technologies: []techutils.Technology{techutils.Npm},
+			expectedOutput: map[string]string{
+				JfPackageManagerEnvVariable: string(techutils.Npm),
+				JfLanguageEnvVariable:       string(techutils.JavaScript),
+				utils.JfMsiEnvVariable:      "msi",
+				gitRepoEnvVariable:          "gitRepoUrl",
+			},
+		},
+		{
+			name:         "With project key",
+			msi:          "msi",
+			gitRepoUrl:   "gitRepoUrl",
```

Comment:
does gitRepoUrl required in the projectKey case?

Reply:
yes